ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,7 +33,6 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
```

Comment:
Revert this change to , i think you need to make all your new changes with in this file https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java, otherwise thanks and keep up